CONTRIBUTING.md

```diff
@@ -15,52 +55,221 @@ simple statement that you, as a contributor, have the legal right to make the
+  to get involved. Take the time to write a proper description and summarise key points.
+1. Take care with formatting. Ensure the [markdown is tidy](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/getting-started-with-writing-and-formatting-on-github),
+  use [code blocks](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks) etc etc.
+  The faster something can be read, the faster it can be dealt with.
+1. Keep it civil. Yes, it is annoying when things don't work, but it is way more fun helping out
+  someone who is not... the worst. Remember that conversing via text exacerbates
+  everyone's negativity bias, so throw in some emoji when in doubt :+1: :smiley: :rocket: :tada:.
+
+**Dedicated guides**:
+- [Bug report guide](#bug-report-guide)
+- [Feature request guide](#feature-request-guide)
+- [Help request guide](#help-request-guide)
+
+## Bug report guide
+
+We hope to get to bug reports within a couple of working days, but please wait for at least
+7 before nudging. (Unless it is a super critical end-of-the world bug, then by all means
+make some noise :loudspeaker:.)
+
+Below are the criteria we like our bug reports to cover in order to gather the bare minimum of
+information. Add more that what is asked for if you can :smiley:.
+
+1. **Search existing issues.** If something similar already exists, and is still open, please contribute to the discussion there.
+
+1. **Write a concise and descriptive title**, like you would a commit message, something which others can easily
+  find when doing step 1 above.
+
+1. **Detail what it was that you were trying to do and what you expected would happen**.
+  Give some background information around what you have already done to your cluster, any custom configuration etc.
+  With sufficient information you can pre-empt any questions others may have. This should cut out some obvious
+  back-and-forth and help get people to the heart of the issue quickly.
+
+1. **Explain what actually happened**. Provide the relevant error message and key logs.
+
+1. **Provide a reproduction**, for example the exact command or a yaml file (sensitive info redacted).
+  Please try to reduce your reproduction to the minimal necessary to help whoever is helping you
+  get to the broken state without needing to recreate your entire environment.
+
+1. **Reproduce the issue with logging verbosity set to at least 4** (`-v=4`), if you have not already done so. Ensure
+  logs are formatted with [code blocks](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks).
+  If they are long (>50 lines) please provide them in a Gist or collapsed using
+  [HTML details tags](https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d).
+  Take care to redact any sensitive info.
+
+1. Paste in the outputs of `eksctl version` and `kubectl version` where relevant, as
```

Comment:
Could we implement a `eksctl info` or something to that affect that prints everything we need? It could dump eksctl version, kubectl version, OS and anything else we might think is useful (admittedly that was all I could think of)

Reply:
🤔  yeh I like that, we had something similar on Garden, which eventually got to be quite elaborate 😂  https://github.com/cloudfoundry/dontpanic

Reply:
Opened a feature request https://github.com/weaveworks/eksctl/issues/3029